### ISSUE TITLE
Review of AMY docs and corresponding UI operations

### ISSUE THREAD

**Issue by elichad** [carpentries/amy#2328]
Dumping all my notes from a thorough read of the AMY docs and corresponding operations on the test server. Many of them are small things and I'll make a PR to fix as many of those as possible. Some probably deserve their own issues.

## Questions:
- [ ] what criteria make an instructor or other Person 'active' vs 'inactive?' Am I right in thinking that people are made inactive manually?
- [ ] under what criteria would you archive a Person?
- [ ] Training requests: Is there any particular reason why we allow matching trainees to multiple TTT events? I expected it'd throw an error but it didn't.

## Outdated things in docs:
- [ ] Organization: Affiliated organizations not used - hide/remove?
- [ ] Membership: Person roles (Trainer missing from docs)
- [ ] Event: URL validation error suggests pointing to a github repo rather than a github.io site
![url_validation_out_of_date](https://user-images.githubusercontent.com/20683271/216090680-22a65e33-0e3e-4c3a-9621-055210b96d7f.png)
- [ ] Add Person to Event: Title and URL are not used - hide/remove?
- [ ] Docs: various references to separate DC/LC/SWC badges/demos/instructors etc
- [ ] Find workshop staff: Badge filter has been changed to a couple of checkboxes

## Miscellaneous issues:
- [ ] Docs: anchor links don't have a proper icon, just a hard-to-see box 
![image](https://user-images.githubusercontent.com/20683271/216082069-1204657f-e159-4e16-9da8-b368e96765bf.png)
- [ ] dropdowns start to 404 if the page is kept open for a long time (5-10 min), displaying "The results could not be loaded." I assume this is a form timeout and possibly deliberate? But it's not that clear that that's what's happening.
- [ ] Documentation missing for some of the Person tabs - Consents and Community Roles
- [ ] New Event: certain tags must be added to match the selected curriculum, but the warning if you don't doesn't tell you which tags are needed. (Also, I wonder if we need those tags to be user-selectable at all or if we can add them automatically?)
- [ ] Docs: "**URL** Enter in the workshop's url (to the GitHub page, not the repo)" could be clearer as often "GitHub page"=repo
- [ ] New Event: Sometimes an extra blank entry appears when entering the first email into 'additional people to contact.' I assume this isn't causing problems though.
![image](https://user-images.githubusercontent.com/20683271/216087338-302ead36-9e03-4a89-8400-dfe9c512e626.png)
- [ ] Banner isn't fixed when scrolling horizontally (but getting this right for all zooms/screen sizes might be tricky?)
![nav_horizontal_scroll](https://user-images.githubusercontent.com/20683271/216091280-71a52834-42dc-42b7-97c7-dbc2955d2d0b.png)
- [ ] Request training form: family name is a required field here even though it's not required in AMY.
- [ ] Request training form: Duplicate text 'This response is optional and doesn't impact your application's ranking.'
- [ ] Find workshop staff: Clarify that selecting lessons will return only instructors who can teach **all** of those lessons

## Accessibility issues:
- [ ] in docs, include symbols for external links & open them in a new tab
- [ ] New event: can't select multiple non-adjacent items in the event tag list with a keyboard (as far as I can tell)
- [ ] Bulk add people: can't tab through the table that's created when bulk-adding people to events (and therefore can't make corrections by keyboard)
- [ ] New event: specify how to separate email addresses in the docs
- [ ] Workshop requests: 'Accept and create event' screen does not work on a narrow screen/at high zoom - the columns overlap
![accept_create_event_w_zoom](https://user-images.githubusercontent.com/20683271/216090828-9af36f47-a67e-4fae-9ec2-bb5858fc637d.png)
- [ ] Request training form: options are styled as links, not sure why
- [ ] Sometimes warnings appear after a form has already been submitted - better if they appear before so changes can be made if needed. (I made this note while playing with the training requests page, but don't remember which warnings I was seeing...)
- [ ] Training requests: Warning about unmatching before rematching shouldn't be a tooltip - it's easy to press the button before you realise the warning is there!
- [ ] Find workshop staff: filter sidebar doesn't display well on a narrow screen. It's not great in general but worse on this page because the location block is nested.
![image](https://user-images.githubusercontent.com/20683271/216118837-9556a1b9-533f-4051-bbf4-656cf307b011.png)

**Comment by maneesha** [comment]
> what criteria make an instructor or other Person 'active' vs 'inactive?' Am I right in thinking that people are made inactive manually?
>  under what criteria would you archive a Person?

These are both part of the soon-to-be-defined archival process; there is no current system.

**Comment by maneesha** [comment]
> Training requests: Is there any particular reason why we allow matching trainees to multiple TTT events? I expected it'd throw an error but it didn't.

Yes, someone can be matched to multiple TTT events if the had to re-take a training.  

>

**Comment by maneesha** [comment]
>  Organization: Affiliated organizations not used - hide/remove?

This is another example of a feature that was implemented but not fully thought through.  We had wanted to be able to identify organizations that are identified with each other through their own networks (not through The Carpentries).  

For example, in the United States, there is the University of California system.  Within that, there are many colleges and universities (University of California Los Angeles, University of California Irvine, University of California San Francisco, etc...).  We wanted to be able to identify all organizations that were part of a given network.

The relationship matching was difficult to enforce.  We never used this feature, and should consider whether we want to keep it.

**Comment by maneesha** [comment]
@elichad  can put in PRs to update these items:  

> -  Membership: Person roles (Trainer missing from docs)
> -  Event: URL validation error suggests pointing to a github repo rather than a github.io site
> -  url_validation_out_of_date
> -  Docs: various references to separate DC/LC/SWC badges/demos/instructors etc
> -  Find workshop staff: Badge filter has been changed to a couple of checkboxes

For this item: 
>  Add Person to Event: Title and URL are not used - hide/remove?

That's correct - we aren't using those fields.  This was brought up for [in this issue](https://github.com/carpentries/amy/issues/2094) which I said I'd start on but never did... this would be a good issue for @elichad

**Comment by maneesha** [comment]
For all of the accessibility issues - we are planning to do a comprehensive accessibility audit of AMY this year.  @elichad  can you open a new issue that is a log of all accessibility issues we come across so we have a good starting point for that?
